src/pages/wishlist/WishList.jsx

```diff
@@ -0,0 +1,39 @@
+import { useDocTitle } from "../../hooks/useDocTitle";
+import "./wishlist.css";
+import { Card } from "../../components";
+import { useAuth, useWishList } from "../../context";
+import { Link } from "react-router-dom";
+
+export function WishList() {
+  useDocTitle("Wishlist - SmashCart - Manoj Sarna");
```

Comment:
Appreciation: This is nice, if you could tell me how this works it would be awesome because i wanna use this as well

Reply:
Yes sure , anytime :)